addons/full-signature/blocks.css

```
@@ -1,3 +1,7 @@
 .scratchblocks {
   overflow: auto;
 }
+
+.sblocks3 {
```

Pretty sure it's `.blocks3`, not `.sblocks3`
```suggestion
.blocks3 {
```